packages/@aws-cdk/cdk/lib/core/tag-manager.ts

```diff
@@ -0,0 +1,157 @@
+import { Construct } from './construct';
+import { Token } from './tokens';
+
+/**
+ * ITaggable indicates a entity manages tags via the `tags` property
+ */
+export interface ITaggable {
+    tags: TagManager,
+}
+
+/**
+ * A single tag property values
+ */
+interface ITagProps {
```

Comment:
Should I separate this into it's own interface? The compiler seemed to fight me with the inline object definition.

Reply:
I don't see why this interface is needed.